### ISSUE TITLE
Error: Symfony \ Component \ HttpKernel \ Exception \ HttpException No message

### ISSUE THREAD

**Issue by rxng** [Qoraiche/laravel-mail-editor#150]
Going to http://localhost:8000/maileclipse/mailables leads to the following error.

<img width="1082" alt="image" src="https://user-images.githubusercontent.com/19649766/116637476-62f83b80-a9a7-11eb-9175-233a87882af9.png">

I have followed all the install instructions, so seems like this is a bug? I am using version 2.2, on Laravel 5.6.40

**Comment by Qoraiche** [comment]
Thanks for reporting @rxng, could you please share the full error stack?

**Comment by rxng** [comment]
> Thanks for reporting @rxng, could you please share the full error stack?

sure thing here you go  https://drive.google.com/file/d/1b2J33JEKbDvjueh8idIoh9GsSlPn2oAa/view?usp=sharing

**Comment by ReeceM** [comment]
Thank you for reporting @rxng 👍

It seems to be an authorization error when looking at the stack trace and going back. Unfortunately there isn't a message added at that point

<img width="1276" alt="Stack trace error" src="https://user-images.githubusercontent.com/2767904/116676966-edd93600-a9a7-11eb-9099-b8c06d49560b.png">

Could you see if the environment config `'maileclipse.allowed_environments'` is set to what environment you are currently in, it might not be local or it could be set differently. You can confirm in the `.env` file of Laravel: `APP_ENV`

It is unfortunate that this is showing the whoops page and not the 403, so it is still a bug in the missing explanation of the reason.

**Comment by rxng** [comment]
sorry @ReeceM I never got a notification of your reply! How/where do I set the  environment config 'maileclipse.allowed_environments' ?

**Comment by ReeceM** [comment]
All good @rxng :+1:

It is in the `config/maileclipse.php` file. The settings look correct, but as you say it isn't working like that then it has got an issue. Thank you for providing the extra information about it. 

Will have to look at why it isn't working correctly for the environment configuration.

**Comment by rxng** [comment]
> All good @rxng 👍
> 
> It is in the `config/maileclipse.php` file. The settings look correct, but as you say it isn't working like that then it has got an issue. Thank you for providing the extra information about it.
> 
> Will have to look at why it isn't working correctly for the environment configuration.

You’re welcome, good luck with it! @ReeceM

**Comment by ReeceM** [comment]
Hi @rxng I have checked the files and also the logic, it seems the only thing that wasn't right was that the error message wasn't clear enough.

Looking further at the stack trace you shared, I noticed that the result from checking the environment is `false`

This line of code: `App::environment(config('maileclipse.allowed_environments', ['local']))`

```
The arguments in your stack trace Arguments
false
403
```

This would indicate that the environment you are running in is not listed in the `allowed_environments`: local, testing, staging.

Did you run the `php artisan laravel-mail-editor:install` command?
